.github/workflows/code-styling.yml

```
@@ -15,9 +19,30 @@ concurrency:
 
 
 jobs:
+
+  setup-module-matrix:
+    runs-on: ubuntu-latest
+    name: Setup the module matrix
+    outputs:
+      matrix: ${{ steps.read-modules.outputs.matrix }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Read in list of mature analysis modules
+        id: read-modules
+        run: |
+          # From input `mature-modules.json`, read in modules which should be styled and save to $GITHUB_OUTPUT
+          echo "matrix=$(jq -cr . < ./.github/workflows/mature-modules.json)" > $GITHUB_OUTPUT
```

Adding an additional comment, removing unneeded (I think) `./` and `.`. Also always `>>` for `GITHUB_OUTPUT`
```suggestion
        run: |
          # From input `mature-modules.json`, read in modules which should be styled, reformat, and save to $GITHUB_OUTPUT
          echo "matrix=$(jq -cr < .github/workflows/mature-modules.json)" >> $GITHUB_OUTPUT
```